gatk

```diff
@@ -37,12 +37,20 @@ EXTRA_JAVA_OPTIONS_SPARK= "-DGATK_STACKTRACE_ON_USER_EXCEPTION=true " \
                    "-Dsamjdk.use_async_io_read_samtools=false " \
                    "-Dsamjdk.use_async_io_write_samtools=false " \
                    "-Dsamjdk.use_async_io_write_tribble=false " \
-                   "-Dsamjdk.compression_level=2 "
+                   "-Dsamjdk.compression_level=2 " \
+                   "-Dspark.executor.cores=1 " \
```

Comment:
`spark.executor.cores`, etc., should not be hardcoded in the `gatk` script like this.